### DISCUSSION TITLE
Open rendering in a separate window

### DISCUSSION THREAD

**Discussion by b2renger** [cables-gl/cables_docs#210] [Feature Requests] [answered]
It would be awesome to open the canvas in a seperate window. 

This would allow to work on two monitors : one for editing and one for visualizing fullscreen, but also to take on small video mapping projects.

I realize this is doable using websockets, but having a feature like this available at the click of one button would be really cool.

**Comment by pandrr** [comment, marked as answer]
we are working on this, you can test it on dev.cables.gl and use the remote viewer in the menu

**Reply by cristianvogel** [reply]
Hi Pandrr.

I was experimenting with this feature, but my patch uses the Microphone ( Audio Input ) module and this seems to not work when remote viewing into a seperate window.

**Reply by steam0r** [reply]
hey, the remote patch will use the microphone of the remote window, it's not transmitting any state data to the new window. any interactivity/input will have to be done in both windows (same goes for mouseclicks for example).

**Comment by b2renger** [comment]
this works really nicely :)

once again kudos to the amazing team !

and thank you for the awesome tool  !

🥇 🥇 🥇

**Comment by wirmachenbunt** [comment]
oh, i missed that. cool, this will be helpful when doing the next project
